apps/gitness/src/framework/hooks/useLogs.ts

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useRef, useState } from 'react'
+import { LivelogLine } from '@harnessio/code-service-client'
+import { ExecutionState } from '../../types'
+
+interface UseLogsReturnType {
+  logs: LivelogLine[]
+}
+
+enum LogsSSEEvent {
+  MESSAGE = 'message'
+}
+
+interface UseLogsInterface {
```

Comment:
add this to a interface file?

Reply:
I am thinking we should only add reusable/common interfaces to the interface file, like we have `.types` file today in some directories. This interface is too specific to this hook. What do you think?